docs/00-manuscripts/ieee/reviews/reviewer4.md

```diff
@@ -89,22 +93,21 @@ Additionally it would mainly cause more guessing and move focus away from the ma
 In general the NRI concept is based on the "Optimized multiple narrow-band reflectance" (OMNBR) approach by [Thenkabail et al. (2000)](thenkabail2000) and also described in [Thenkabail et al 2018](https://www.taylorfrancis.com/books/hyperspectral-indices-image-classifications-agriculture-vegetation-prasad-thenkabail-john-lyon-alfredo-huete/e/10.1201/9781315159331).
-One reasons to use the NDVI base as the index for NRI calculation was the availability in the R package [{hsdar}](https://cran.r-project.org/web/packages/hsdar/index.html) in the function [nri()](https://rdrr.io/cran/hsdar/man/normalized.ratio.index.html) which uses the NDVI-index as the base.
-The use of NRI indices was thought as an addition to the existing feature sets (HR and VI) and using only a generic NDVI formula as the base was sufficient in our view.
+One reason to use the NDVI-type index formulation for NRI calculation was its availability in the R package [{hsdar}](https://cran.r-project.org/web/packages/hsdar/index.html) in the function [nri()](https://rdrr.io/cran/hsdar/man/normalized.ratio.index.html). Other indices, e.g. obtained by simple ratioing, would be expected to be very strongly correlated with the corresponding NRIs. Corresponding the already remarkably high dimensionality of feature space, it would be less likely that such further enhancement of feature space would lead to any improvements. Our data and code are of course available for follow-up studies that might focus on different feature extraction strategies.
 
 > - The RMSE alone is not a good quality measure, since it is highly dependent on the variance of the sample (as the authors describe themselves in the manuscript). Please supplement this measure with another one, e.g. R² (but there are more sophisticated ones around) to evaluate the quality of your results.
 
 FIXME: schauen was bei r-sq im neuen run raus kommt.
 Thanks for suggesting to take another look at the chosen measure(s).
-We have chosen RMSE as it is defined in the unit of the response variable and the resulting value can be interpreted in a somewhat meaningful way.
-In contrast r-sq shows how much variance of the response variable has been explained by the model and might be more complex to interpret in the current case.
-Nevertheless we have added the r-sq values for the best performing models to supplement the RMSE measure.
+We have chosen RMSE as it is defined in the unit of the response variable and the resulting value can therefore be interpreted by domain experts.
+In contrast the $R^2$ shows how much variance of the response variable has been explained by the model and might be more complex to interpret in the current case. It may also be problematic to average $R^2$ values across multiple cross-validation test sets since each of them has a different variance of the response variable, which changes the denominator of $R^2$ between test sets.
```

Comment:
```suggestion
In contrast the $R^2$ shows how much variance of the response variable has been explained by the model and might be more complex to interpret in the current case.
It may also be problematic to average $R^2$ values across multiple cross-validation test sets since each of them has a different variance of the response variable, which changes the denominator of $R^2$ between test sets.
```

> It may also be problematic to average $R^2$ values across multiple cross-validation test sets since each of them has a different variance of the response variable, **which changes the denominator of $R^2$ between test sets**.

I do not really understand this part - maybe we can discuss this in a call at some point :)